MEETINGS.md

```diff
@@ -1,26 +0,0 @@
-# Meeting Dates 
```

Comment:
So, Git thinks I deleted this (and the other files in `meeting/`, but they have actually just been moved, so the diffs are all horrible